website/src/components/pageStateSelectors/WasapPageStateSelector.tsx

```diff
@@ -402,3 +390,46 @@ function LabeledField({ label, children }: { label: string; children: React.Reac
         </label>
     );
 }
+
+function NumericInput({
+    label,
+    value,
+    min,
+    max,
+    step,
+    onChange,
+    parser = parseFloat,
+}: {
+    label: string;
+    value: number;
+    min: number;
+    max: number;
+    step: number;
+    onChange: (v: number) => void;
+    parser?: (v: string) => number;
+}) {
+    return (
+        <LabeledField label={label}>
+            <div className='mb-2 w-full'>
+                <input
+                    className='input input-bordered mb-2 w-full'
+                    type='number'
+                    min={min}
+                    max={max}
+                    step={step}
+                    value={value}
+                    onChange={(e) => onChange(parser(e.target.value))}
```

Comment:
I think this also does the job, then we would not need the `parser`. For the min count input, we could maybe add `setPageState({ ...pageState, minCount: Math.round(v) })` (or we don't care that someone can input floats into the text field).
```suggestion
                    onChange={(e) => onChange(Number(e.target.value))}
```